common/persistence/src/test/java/app/coronawarn/server/common/persistence/domain/DiagnosisKeyTest.java

```diff
@@ -87,4 +88,30 @@ void testRetentionThresholdRejectsNegativeValue(int daysToRetain) {
     assertThatIllegalArgumentException()
         .isThrownBy(() -> diagnosisKey.isYoungerThanRetentionThreshold(daysToRetain));
   }
+
+
+  //Extra tests by Hallon, follows grammar above.
+
+  @Test
+  void testNullDoesNotEqual() {
+    assertThat(diagnosisKey.equals(null).isFalse);
+  }
+  @Test
+  void testSameObjectReturnsTrue() {
+    assertThat(diagnosisKey.equals(this));
```

Comment:
Same here, this should be `assertThat(diagnosisKey).isEqualTo(this)` (although I think that should always be false).